Sources/xcbeautify/Xcbeautify.swift

```diff
@@ -35,6 +35,12 @@ struct Xcbeautify: ParsableCommand {
     @Option(help: "The name of JUnit report file name")
     var junitReportFilename = "junit.xml"
 
+    @Flag(help: "Tell the renderer to treat warnings as errors (github-actions only)")
+    var renderWarningsAsErrors = false
+
+    @Flag(help: "Tell the renderer to skip annotations for warnings (github-actions only)")
+    var renderQuietWarnings = false
```

Comment:
These 2 are related (e.g. it wouldn't make sense to set both to true), right? In that case it would probably be better to have a single parameter, or at least we should validate it